lib/usd/ui/USDImportDialog.cpp

```diff
@@ -71,6 +71,7 @@ USDImportDialog::USDImportDialog(const std::string& filename, const ImportData*
 	fUI->treeView->setAlternatingRowColors(true);
 	fUI->treeView->setSelectionMode(QAbstractItemView::SingleSelection);
 	QObject::connect(fUI->treeView, SIGNAL(clicked(const QModelIndex&)), this, SLOT(onItemClicked(const QModelIndex&)));
+	QObject::connect(fUI->actionReset_File, SIGNAL(triggered(bool)), this, SLOT(onResetFileTriggered()));
```

Comment:
Hooking up the menu item action to the reset function.  It's not a check-box menu so I'm ignoring the bool state variable.